features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -2137,6 +2137,13 @@
         <!--Timeout in milliseconds for the waiting external calls-->
         <LongWaitTimeout>10000</LongWaitTimeout>
 
+        <!--Timeouts for callChore token call and adaptive endpoint-->
+        <!--Default timeout in milliseconds-->
+        <HTTPConnectionTimeout>1000</HTTPConnectionTimeout>
+        <HTTPReadTimeout>1000</HTTPReadTimeout>
+        <HTTPConnectionRequestTimeout>1000</HTTPConnectionRequestTimeout>
```

Comment:
According to the j2 file, this should come inside `<CallChoreo>`

Reply:
Fixed with 92e3c8402c07c18add6ceaec8bf0294f8bbae456